corehq/apps/users/templates/users/enterprise_users.html

```diff
@@ -16,14 +16,19 @@
           <h3 class="panel-title" style="padding-top: 7px;">{% trans 'Users' %}</h3>
         </div>
         <div class="col-sm-6">
-          <search-box data-apply-bindings="false"
-                      params="value: query,
-                              action: function() { goToPage(1); },
-                              placeholder: '{% trans_html_attr "Search Users..." %}'"></search-box>
+            <button type="button" class="btn btn-default pull-right "
```

Comment:
Optional suggestion: you could just use one button instead of two you made the click function `showDeactivated(!showDeactivated())` and made the text a computed observable that checks the value of `showDeactivated`.